benchmark/__main__.py

```diff
@@ -0,0 +1,20 @@
+"""Run the benchmarks for the given scenario."""
+
+import logging
+
+from benchmark.domain import BENCHMARKS
+from benchmark.result.result import Result
+
+
+def main():
+    """Run the performance test for the given scenario."""
+    for benchmark in BENCHMARKS:
+        benchmark_result: Result = benchmark.run()
+        logging.info(
```

Comment:
We already discussed this, but could you remind me why the logging is done here? Was it for more verbosity in the pipeline?

Reply:
Yes, to have some visual feedback while the benchmarks are executed. We may remove it because execution feedback is provided by Babye itself.